alembic/versions/2022-02-21-19.56.24_6b0f5f32d602_initial_tables.py

```diff
@@ -278,6 +279,7 @@ def upgrade():
         sa.Column("group_id", mealie.db.migration_types.GUID(), nullable=False),
         sa.Column("name", sa.String(), nullable=False),
         sa.Column("slug", sa.String(), nullable=False),
+        sa.Column("color", sa.String(), nullable=False),
         sa.ForeignKeyConstraint(
             ["group_id"],
```

Comment:
These files are readonly. You can't update an existing migration, you need to generate a new one.

Reply:
Thanks a lot for your feedback! I have one question. How do you generate these read-only files?

Reply:
You'll want to generate a migration using [alembic](https://alembic.sqlalchemy.org/en/latest/autogenerate.html).

Here's an example:
```alembic revision --autogenerate -m "add color to tags and categories"```

(that might be all you have to do for this migration, you can adjust the message)

Your new auto-generated file will have a new `Revision ID` at the top, which you'll want to put in the alembic test file (if you try running pytest it will fail on the alembic test if you don't update it)

Reply:
hi there! thanks a lot for your response. I generated a new migration using alembic and replaced the version_num with the new Revision ID.

Reply:
@dorj222, CI is still failing. I believe there have been some migrations since you've started this PR. Migrations _need_ to happen in a specific order so you will likely need to rebase the PR and get all the changes from the latest branch and update your migration revision information to reflect the correct order.